### PR TITLE
Onboarding steps for activating a Display

### PR DESCRIPTION
**Pull request by alex-deaconu**
Add onboarding bar to main page
Hide onboarding until launcher data is loaded
Only load launcher/onboarding data once

If onboarding is not complete, add event handlers to check creation/updates

[stage-3]

### CODE REVIEW

**Comment by alex-deaconu** [comment]
@ezequielc @fjvallarino please review. Thanks!

File: test/unit/launcher/services/svc-launcher-factory.tests.js [hunk 1]
```diff
@@ -0,0 +1,180 @@
...
+    presentation.list = sinon.spy(function() {
+      presentationListDeferred = Q.defer();
+
+      return presentationListDeferred.promise;
+    });
+
+    canAccessDeferred.resolve();
+    
+    setTimeout(function() {
+      expect(launcherFactory.presentations).to.deep.equal({loadingItems: true, list: []});
```
Comment (fjvallarino) [inline test/unit/launcher/services/svc-launcher-factory.tests.js]: @alex-deaconu I'm not sure I understand this one. Based on the name of the test, shouldn't _loadingItems_ be false at this point?

Comment (alex-deaconu) [inline test/unit/launcher/services/svc-launcher-factory.tests.js]: @fjvallarino this one is a bit confusing. The order of execution is the following: 

`launcherFactory.load()` is initialized and the promise is the final step of the test (see `done();`)

`presentation.list` is mocked and the promise saved

After `canAccessDeferred.resolve();` and the timeout you expect the displays and schedules calls to be resolved, but the presentations promise is still not.

Once the `presentationListDeferred.resolve` is called, the original promise for `load()` is executed and `done();` is called.

Let me know if this clears things up. Thanks!

Comment (fjvallarino) [inline test/unit/launcher/services/svc-launcher-factory.tests.js]: @alex-deaconu sounds good

**Comment by fjvallarino** [comment]
@alex-deaconu looks good! :+1:

**Pull request merged**